bump_dictionary/cli.py

```diff
@@ -0,0 +1,113 @@
+import json
+from pathlib import Path
+
+import typer
+from typing_extensions import Annotated
+
+from . import utils
+from .logger import VerbosityLevel, configure_logger, log_error, logger
+from .models import latest_dictionary_model, legacy_dictionary_model
+
+bump_dictionary = typer.Typer(
+    help="Bump Neurobagel data dictionaries to the latest version of the data dictionary schema.",
```

Comment:
is it `latest` and `previous` or `current` and `legacy`? I'll just comment here but we need to pick one and stick to it - also in code. Otherwise we confuse ourselves too

Reply:
I think I like `legacy` and `latest` best - will standardize it in the codebase.

Lmk if you strongly prefer an alternative